File: 강재연/programmers/P181865.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181865
// 시간: 0.07 ms
// 메모리: 33.4 MB

function solution(binomial) {
    const [a, x, b] = binomial.split(' ');
    switch(x){
        case '+' : return +a + +b;
        case '-' : return +a - +b;
        case '*' : return +a * +b;        
    }
}

File: 공예진/programmers/P181865.java
//문제: https://school.programmers.co.kr/learn/courses/30/lessons/181865
//시간: 0.16ms
//메모리: 75.9MB

class Solution {
    public int solution(String binomial) {
        int answer = 0;
        String[] arr = binomial.split(" ");
        int a = Integer.parseInt(arr[0]);
        int b = Integer.parseInt(arr[2]);
        switch (arr[1]) {
            case "+" : answer = a + b; break;
            case "-" : answer = a - b; break;
            case "*" : answer =  a * b; break;
        }
        return answer;
    }
}